brands/shop/convenience.json

```diff
@@ -1594,7 +1594,7 @@
     ],
     "tags": {
       "brand": "Rewe To Go",
-      "brand:wikidata": "Q16968817",
+      "brand:wikidata": "Q85224313",
       "brand:wikipedia": "en:REWE",
```

Comment:
If you are changing the `brand:wikidata` to a better QID, please remove the `brand:wikipedia` tag, which still points to the old REWE Wikipedia page.  Ideally there will eventually be a separate Wikipedia page for REWE To Go.